v22.1/security-reference/transport-layer-security.md

```diff
@@ -0,0 +1,280 @@
+
+- Learn about [Managing CockroachDB security certificates with HashiCorp Vault](../manage-certs-vault.html).
+- Review our [breakdown of security features by offering](security-overview.html).
+- Contact our <a href="mailto:sales@cockroachlabs.com">sales team</a> to discuss your needs and the range of solutions offered by Cockroach Labs.
+{{site.data.alerts.end}}
+
+By default, a CockroachDB node makes double use of a single key pair, using it to authenticate both as client when initiating a connection to another node, and as server when recieving requests.
+
+The node must also have a trust store containing the public certificate of at least one trusted CA&mdash;the one that issued the public certificates of any nodes it must connect with.
+
+## TLS in CockroachDB SQL client connections
+
+CockroachDB provides a number of SQL clients, including a [CLI](../cockroach-sql.html#start-a-sql-shell), and a number of [drivers and object-relational mapping (ORM) tools](../install-client-drivers.html). Regardless of which client you are using, how you are able to authenticate to a CockroachDB cluster depends on that cluster's [authentication configuration](authentication.html), specifically whether that configuration requires the user to authenticate with username/password combination, certificate or another method.
+
+In turn, which authentication methods are available depends on the sort of environment in which a CockroachDB cluster is deployed, as described in the following.
+
+### CockroachDB Cloud
+
+{{ site.data.products.db }} currently does not support certificate-authenticated client requests. TLS is still used to authenticate the server and encrypt all traffic, but the user must authenticate to the database with a username/password combination.
+
+Because the server must still be TLS authenticated, the client must know to trust the certificate authority that signed the public certificate identifying the server. The path to the CA's public certificate is passed as the `sslrootcert` parameter in a [database connecton string](../connect-to-the-database.html), or by being placed in the directory specified by the `certs-dir` argument in a connection made with the [`cockroach sql`](../cockroach-sql.html) CLI command.
+
+### Self-Hosted CockroachDB
+
+{{site.data.alerts.callout_info}}
+Customers who deploy and manage their own CockroachDB clusters must provision and manage certificates on each nodes, implementing their own PKI security. This entails ensuring that credentials are carefully controlled, monitoring for signs of compromise and mitigating the impact of potential credential leaks. Authorization for issuing credentials is particularly critical, and this includes issuing private key/public certificate pairs for CockroachDB nodes or clients. Unmitigated compromise of either of these can have devastating business impact.
```

Comment:
"each nodes" Also "This entails ensuring" is a bit circuitous. Maybe "their own PKI security by: [list each action]"